Dockerfile

```diff
@@ -1,42 +1,32 @@
 ARG ONOS_BUILD_VERSION=stable
-FROM debian:stretch
+FROM golang:1.11-alpine
 LABEL maintainer="Sean Condon <sean@opennetworking.org>, Adib Rastegarnia <adib@opennetworking.org> "
 LABEL description="Builds a gNMI/gNOI simulator on a Debian distribution"
 
+RUN apk add --update bash && rm -rf /var/cache/apk/*
 
-
-RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -qy --no-install-recommends \
-      apt-utils \
+RUN apk update \
+    && apk add \
       ca-certificates \
-      software-properties-common \
-    && echo "deb http://deb.debian.org/debian stretch-backports main" >  /etc/apt/sources.list.d/backports.list \
-    && apt-get update
+    && apk update
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -qy --no-install-recommends \
-    fping \
+RUN apk add \
+    #fping \
     git \
-    golang-1.11-go \
     iproute2 \
-    iputils-ping \
+    iputils \
     net-tools \
     netcat-openbsd \
     openssh-client \
     psmisc \
     procps \
-    sudo \
-    vim
+    sudo 
+    #vim
 
 ENV HOME=/home/devicesim
 RUN mkdir $HOME
 WORKDIR $HOME
 
-ENV GOROOT=/usr/lib/go-1.11
```

Comment:
they are probably in the image already

Reply:
Yes. I should check and see if they are not needed, I will remove them.